src/usr/local/www/widgets/widgets/thermal_sensors.widget.php

```diff
@@ -31,10 +31,10 @@
 
 	$specplatform = system_identify_specific_platform();
 	if ($specplatform['name'] == 'SG-5100') {
-		$_gb = exec("/sbin/sysctl -q dev.cpu | grep temperature | sort",
+		$_gb = exec("/sbin/sysctl -q dev.cpu | /usr/bin/grep temperature | sort",
```

Comment:
May as well use the same pattern here, too. Plus add the full path to `sort`.

Reply:
Done.